lib/ansible/modules/tacp/tacp_info.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2020, Lenovo
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.tacp_ansible import tacp_utils
+
+
+import json
+import tacp
+import sys
+from tacp.rest import ApiException
+from pprint import pprint
```

Comment:
Looks like you can clean these imports. `json`, `pprint`, and `sys` are not being used.

Reply:
done! good catch :)